conf/pawsey_setonix.config

```diff
@@ -0,0 +1,29 @@
+// Pawsey Setonix nf-core configuration profile
+params {
+  config_profile_description = 'Pawsey Setonix HPC profile provided by nf-core/configs'
+  config_profile_contact = 'Sarah Beecroft (@SarahBeecroft), Georgie Samaha (@georgiesamaha)'
+  config_profile_url = 'https://support.pawsey.org.au/documentation/display/US/Setonix+User+Guide'
```

Comment:
Missing the `max_*` parameters (which would solve my comment below about the queues)

Reply:
Added